src/Spice86/Models/Debugging/CpuStateInfo.cs

```diff
@@ -0,0 +1,84 @@
+namespace Spice86.Models.Debugging;
+
+using CommunityToolkit.Mvvm.ComponentModel;
+
+using System.ComponentModel;
+
+public partial class StateInfo : ObservableObject {
+    [ObservableProperty] private byte _AH;
+    [ObservableProperty] private byte _AL;
+    [ObservableProperty, Category("Flags")] private bool _auxiliaryFlag;
+    [ObservableProperty] private ushort _AX;
+
+    [ObservableProperty] private byte _BH;
+    [ObservableProperty] private byte _BL;
+
+    [ObservableProperty] private ushort _BP;
+    [ObservableProperty] private ushort _BX;
+    [ObservableProperty, Category("Flags")] private bool _carryFlag;
+
+    [ObservableProperty] private byte _CH;
+    [ObservableProperty] private byte _CL;
+
+    [ObservableProperty, Category("Flags")] private bool? _continueZeroFlag;
+
+    [ObservableProperty] private ushort _CS;
+    [ObservableProperty] private ushort _CX;
+
+    [ObservableProperty, ReadOnly(true)] private long _cycles;
+
+    [ObservableProperty] private byte _DH;
+
+    [ObservableProperty] private ushort _DI;
+
+    [ObservableProperty] private short _direction16;
```

Comment:
Not sure direction is needed in the UI, it is a derived value from _directionFlag

Reply:
Editing them directly does not make sense ?

Reply:
No, and in fact they are calculated properties:
```
public short Direction8 => (short)(DirectionFlag ? -1 : 1);
```

Reply:
Fixed.